src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -287,12 +288,14 @@ def _export_flow_connections(
     ):
         from promptflow.contracts.flow import Flow as ExecutableFlow
 
-        executable = ExecutableFlow.from_yaml(flow_file=Path(flow_dag_path.name), working_dir=flow_dag_path.parent)
+        executable = ExecutableFlow.from_yaml(flow_file=Path(flow_dag_path.name),
+                                              working_dir=flow_dag_path.parent.absolute())
 
-        return self._migrate_connections(
-            connection_names=executable.get_connection_names(),
-            output_dir=output_dir,
-        )
+        with _change_working_dir(flow_dag_path.parent):
```

Comment:
Will there be a case that it used to work but won't work after this change? You changed the woring dir, is there a case that this `migrate_connection` just need the working dir to be original `flow_dag_path.parent`?

Reply:
Should not cause other problems. If there is strongly associated with the original path in user code, it will caused the executor failed.